tools/logger/CMakeLists.txt

```diff
@@ -1,5 +1,7 @@
 # SPDX-License-Identifier: BSD-3-Clause
 
+cmake_minimum_required(VERSION 3.10)
```

Comment:
Why did you add it? It's needed usually in top-level file (in this case tools).

Reply:
Ok I see in commit message now. Anyway it ecnourages users to build logger as standalone tools (instead of being part of tools) IDK if we want to support both cases, better to manage it from one place.

Reply:
I didn't notice that and use it as top level one. Deleted.